Anytype/Sources/PresentationLayer/Modules/Discussion/Container/DiscussionSpacingContainer.swift

```diff
@@ -0,0 +1,29 @@
+import Foundation
+import SwiftUI
+
+struct DiscussionSpacingContainer<Content: View>: View {
+    
+    let content: Content
+    @Environment(\.anytypeNavigationPanelSize) var navigationSize
+    
+    init(@ViewBuilder content: () -> Content) {
+        self.content = content()
+    }
+    
+    var body: some View {
+        GeometryReader { reader in
+            GeometryReader { readerIgnoreKeyboard in
+                content
```

Comment:
Naming is a bit obscure:
reader have `.ignoresSafeArea(.**keyboard**, edges: .bottom) and `
`readerIgnoreKeyboard` have ` .ignoresSafeArea(.**all**, edges: .bottom)`
Can you please make it more obvious what is happening here via naming or at least comments? 🙏

Reply:
`readerWithAllArea` and `readerWithoutKeyboardArea` is better?

Reply:
great!